gui/widgets/grid.h

```diff
@@ -222,6 +222,8 @@ class GridWidget : public ContainerWidget, public CommandSender {
 
 	void setSelected(int id);
 	void setFilter(const Common::U32String &filter);
+
+	static const int kThumbGfxMargin = 5;
```

Comment:
Why not stick this into the theme as a parameter, similar to other such variables?

Reply:
I just did it.
The main limitation is that draw data is specific to theme while variables are defined in layout (which is shared among several themes).
If a theme wants to change the drawdata of the grid item background, it will need its own layout.

On another matter, I don't know if this needs a version update. Not having the parameter in theme just makes it look like before the fix.